src/api/contracts/superhero.ts

```diff
@@ -0,0 +1,62 @@
+export interface Powerstats {
```

Comment:
maybe it will be useful to create class entities to encapsulates some logic then there will be a possible to `implements` the interfaces. Also, types are may be sometimes less performant while transplied.